src/commands/misc/lorem.js

```diff
@@ -0,0 +1,29 @@
+const { SwitchbladeEmbed, Command, CommandError } = require('../../')
+const { LoremIpsum } = require('lorem-ipsum')
+
+module.exports = class Lorem extends Command {
+  constructor (client) {
+    super({
+      name: 'lorem',
+      parameters: [{
+        type: 'number',
+        full: true,
+        missingError: 'commands:lorem.noNumber',
+        required: true
```

Comment:
Dont think this should be required. Perhaps default it to 1 or 2 instead?

Reply:
Two words?

Reply:
Oh, okay, done :)